src/main/java/picocli/CommandLine.java

```diff
@@ -14751,9 +14751,9 @@ private BuiltIn() {} // private constructor: never instantiate
                 description = "Show usage help for the help command and exit.")
         private boolean helpRequested;
 
-        @Parameters(paramLabel = "COMMAND", descriptionKey = "helpCommand.command",
+        @Parameters(paramLabel = "COMMAND", arity="0..1", descriptionKey = "helpCommand.command",
                 description = "The COMMAND to display the usage help message for.")
-        private String[] commands = new String[0];
+        private String command;
```

Comment:
Let's keep the name `commands` unchanged, and only change the type from `String[]` to `String`.

That will allow previously generated `reflect.json` files to continue to work with both older and newer versions of picocli.

Reply:
Of course, I'll be reviewing my copy of Plural Logic book! ;-)
What is more would it have been possible to keep String[] and just set the arity?
I did not try.